### PR TITLE
feat: enable hmr by default for new projects

### PR DESCRIPTION
**Pull request by Fatme**
Implements: https://github.com/NativeScript/nativescript-cli/issues/4394

## PR Checklist

- [ ] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [ ] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [ ] You have signed the [CLA](http://www.nativescript.org/cla).
- [ ] All existing tests are passing: https://github.com/NativeScript/nativescript-cli/blob/master/CONTRIBUTING.md#contribute-to-the-code-base
- [ ] Tests for the changes are included.

### CODE REVIEW

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

File: lib/options.ts [hunk 1]
```diff
@@ -21,6 +21,35 @@ export class Options {
...
+			this.argv.hmr = !this.argv.release;
+		}
+
+		// --no-hmr -> hmr: false or --hmr false -> hmr: 'false'
+		const noHmr = parsed && (parsed.hmr === false || parsed.hmr === 'false');
+		if (noHmr) {
+			this.argv.hmr = false;
+		}
+
+		if (noBundle) {
```
Comment (KristianDD) [inline lib/options.ts]: Maybe add a warning for the user that we will override both hmr and bundle options.

**Pull request merged**